webapp/lib/app/nook/controller_view_helper.dart

```diff
@@ -144,39 +144,27 @@ void _populateReplyPanelView(List<model.SuggestedReply> replies) {
   }
 }
 
-void _populateTagPanelView(List<model.Tag> tags) {
+void _populateTagPanelView(Map<String, List<model.Tag>> tagsByGroup, bool showShortcut, Set<String> tagIds) {
```

Comment:
nit: `tagIds` doesn't describe very well what this parameter is used for
```suggestion
void _populateTagPanelView(Map<String, List<model.Tag>> tagsByGroup, bool showShortcut, Set<String> mandatoryExcludeTagIds) {
```